src/_pytest/helpconfig.py

```diff
@@ -170,6 +170,8 @@ def showhelp(config: Config) -> None:
         help, type, default = config._parser._inidict[name]
         if type is None:
             type = "string"
+        if help in (None, ""):
```

Comment:
I think `""` should be accepted, i.e. just make it not crash in this case. It's not very nice but I don't see a particular reason to prevent it if someone really wants to pass that. Further, the same issue happens with `"  "` (two spaces) etc..

Reply:
What would the benefit of a help command with no text be though @bluetech? I don't see a definite reason to ban it, but at the same time I don't see a reason to accept it.

Regarding arbitrary empty spaces, we could just strip the space off of help, if it's possible to retrieve such a value in the first place. ( @hp310780 can you confirm please )

Reply:
> but at the same time I don't see a reason to accept it.

The reason is just to avoid extra code. There are many unreasonable things a user can pass, like a 1000 word explanation, or writing it all in upper case, but we can't check them all. So I think it's better to just accept what the user specifies, as long as it has the right type.

Reply:
I agree @bluetech

Reply:
Okay, so this would accept any string (including the empty string) and only raises an error on `None`?

Reply:
I think that’s what @bluetech had in mind. The idea is we should never guess on behalf of the user, since we can never really exhaustively guess everything the user may do